xilem/examples/calc.rs

```diff
@@ -184,16 +184,20 @@ impl Calculator {
     }
 }
 
+// TODO: Is there a more generic way of doing the return type? This doesn't allow different
+//      "types" of grid_pos values. For example, I couldn't add the operator button.
+fn num_row(nums: [&'static str; 3], row: i32) -> Vec<GridItem<impl WidgetView<Calculator>, Calculator, ()>> {
```

Comment:
I'd expect this to return `impl GridSequence`

Reply:
Changed. That works.